cmd/dep/godep_importer.go

```diff
@@ -17,7 +17,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-const godepJSONName = "Godeps.json"
+const (
```

Comment:
Since we only ever use those two variables by joining them together, we could simpify further to just `godepPath = "Godeps/Godeps.json"` or something